src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,59 @@
+import { Heading, Input, Text, Box, Button, Card } from "@twilio-paste/core";
+import { useEffect, useState } from "react";
+import { RgbaStringColorPicker } from "react-colorful";
+
+interface IColorPickerProps {
+  color: string;
+  title: string;
+  onChange?: React.Dispatch<React.SetStateAction<string>>;
+  colorChanged: (newColor: string) => void;
+  //setDefault?: () => void;
+}
+
+export const ColorPicker = (props: IColorPickerProps) => {
+  const [color, setColor] = useState<string>(props.color);
+
+  useEffect(() => {
+    props.colorChanged && props.colorChanged(color);
```

Comment:
None that I found unfortunately, agree its not ideal. but is a sympto of the implementation of the react-colorful components onChange being  of type `React.Dispatch<React.SetStateAction<string>>` rather than `React.ChangeEventHandler<HTMLInputElement>` and the way I needed to implement in ThemeConfigurationPanel.tsx